tools/CMakeLists.txt

```diff
@@ -1,17 +1,33 @@
-add_kissfft_executable(fastconvr kiss_fastfir.c kiss_fftr.c)
+add_kissfft_executable(fastconvr kiss_fastfir.c)
```

Comment:
just so I understand ... `kiss_fftr.c` is no longer required here because it is incorporated above, right?